Readers/FastqLineReader.cs

```diff
@@ -61,10 +61,10 @@ public bool ReadSequence(out Sequence sequence)
                     return false;
                 }
 
```

Comment:
hmm.... an empty read here really should blow up

Reply:
like throw an `EndOfStreamException`?